<?xml version="1.0" encoding="UTF-8"?>  
<launch>

<!--camera_frame_broadcaster-->
    <node pkg="sendEstimateData" type="camera_frame_broadcaster" name="camera_frame_broadcaster" />

<!--aruco_listener-->
    <node pkg="sendEstimateData" type="aruco_posesSend"    name="aruco_posesSend" output="screen" />

  </launch> 
